_data/projects/cloud-download-manager.yml

```diff
@@ -0,0 +1,17 @@
+---
+name: Cloud Download Manager
+desc: Cloud Download Manager is a useful tool for anyone who wants to store, access, and stream their digital content from the cloud.
+site: https://github.com/besoeasy/Cloud-Download-Manager
+tags:
+- torrent
+- cloud
+- aria2
+- telegram
+- downloader
+upforgrabs:
+  name: up-for-grabs
+  link: https://github.com/besoeasy/Cloud-Download-Manager/labels/up-for-grabs
```

Comment:
🤔 this label doesn't exist in the project, and it's not clear to me which tasks in the repository would belong to this label. Are you able to craft some tasks and ensure this link points to the list of tasks?